### PR TITLE
Adding web packaging

### PR DESCRIPTION
**Pull request by adamroach**

### CODE REVIEW

**Comment by ekr** [comment]
I've looked at this and I think there are major architectural concerns (see my recent message to HTTP). I haven't seen any compelling use cases that require this kind of major architectural change. Accordingly. I think we should mark this either "harmful" or "defer"

**Comment by jyasskin** [comment]
Please don't call this "webpack". That's https://webpack.js.org/.

There's a collection of specifications that were called "web packaging", but they've been split up into at least "signed exchanges" at https://tools.ietf.org/html/draft-yasskin-http-origin-signed-responses-02 and "bundles" or maybe "WebZip" which are draftily drafted at https://github.com/WICG/webpackage/pull/98.

It probably makes sense to add only "signed exchanges" right now, of course with whichever position @ekr and the rest of Mozilla negotiate.

**Comment by dbaron** [comment]
We should keep the discussion about what our position is in #29.  That's the thing that will be referenced from the main table.

That said, I think it's worth answering @jyasskin's question about whether the item we're listing is the right one.  @adamroach - should we still go ahead with the pull request as-is, or does that make you want to list things differently?

(Also, maybe change the title of the PR if you can?)

**Comment by adamroach** [comment]
>That said, I think it's worth answering @jyasskin's question about whether the item we're listing is the right one. @adamroach - should we still go ahead with the pull request as-is, or does that make you want to list things differently?

I'm not entirely certain what else I would point at in this regard. Are you proposing that we point to draft-yasskin-http-origin-signed-responses instead?

**Comment by jyasskin** [comment]
[draft-yasskin-http-origin-signed-responses](https://tools.ietf.org/html/draft-yasskin-http-origin-signed-responses) is my suggestion.

**Comment by dbaron** [comment]
@adamroach did you want to update this?

**Comment by adamroach** [comment]
@dbaron -- I was planning on doing so after https://github.com/mozilla/standards-positions/issues/51 landed. I'd rather the disposition be from the new set rather than the old.

**Comment by dbaron** [comment]
@adamroach ok, just (1 minute ago) landed that

File: activities.json [hunk 1]
```diff
@@ -1,4 +1,15 @@
...
+    "mozBugUrl": null, 
+    "mozPosition": "harmful", 
+    "mozPositionDetail": "Mozilla has concerns about the shift in the web security model required for handling web-packaged infformation. Specifically, the ability for an origin to act on behalf of another without a client ever contacting the authoritative server is worrisome, as is the removal of a guarantee of confidentiality from the web security model (the host serving the webpackage has access to plaintext). We recognise that the use cases satisfied by web packaging are useful, and would be likely to support an approach that enabled such use cases so long as the foregoing concerns could be addressed.",
```
Comment (jyasskin) [inline activities.json]: spelling: "infformation"

Comment (adamroach) [reply activities.json]: Thanks.

**Comment by dbaron** [comment]
In some cases I'll admit I've merged my own PRs after getting review from others.  But today I merged yours promptly, at least.

**Comment by adamroach** [comment]
Thanks!

**Pull request merged**